documents/policies/external/Competition Requirements.md

```diff
@@ -94,96 +115,124 @@ The WCA Delegate must submit the following information when requesting approval
+      4. If the registration requirements cannot be fulfilled by an individual due to reasons outside of their control, they are eligible for completing their registration following a suitable alternative approach, at the discretion of the Delegate. In case of such an inquiry, the organization team must involve the Delegate, who must then carefully assess the individual's situation. If deemed appropriate, the Delegate and organization team must make reasonable efforts to provide an alternative approach for the individual to fulfill the requirements.
          1. Possible reasons include, but are not limited to:
             1. Having an unsupported credit card for paying the registration fee.
          2. Exceptions must not be based on:
             1. Missing the registration deadline.
             2. Not having enough money to pay the registration fee.
+            3. Not being available at the time registration opens.
    2. The registration requirements must include the registration period(s) with a clearly defined time range with a start and end time and date.
       1. The first registration period must be announced publicly on the WCA website at least 48 hours before it starts.
          1. An active registration period may be extended after request of the Delegate.
          2. The competition's WCA Delegate(s) and/or members of the organization team may register for the competition prior to the first registration period, however their registration(s) must not be approved until the first registration period has started.
+            1. The number of WCA Delegates and members of the organization team registered for the competition prior to the first registration period must not exceed 20% of the competitor limit.
       2. Additional registration periods may be added at least 24 hours before they start.
+        1. A registration period may be extended and active immediately, at the discretion of WCAT. 
       3. Different registration periods may have different registration requirements (e.g. restricting which events a competitor can register for).
    3. The registration requirements must include the procedure for completing a valid registration.
-   4. The registration requirements must include the deadline for updating registered events for competitors with completed registrations.
+   4. The registration requirements must include the deadline for adding registered events for competitors with completed registrations.
       1. This deadline must not be earlier than the time registration is set to close. 
-         1. The deadline may be set to be just before an event is scheduled to take place. 
+         1. The default deadline is the same as the time at which registration closes.
+         2. The deadline for adding registered events may be left blank, in which competitors may add registered events up until an event takes place.
```

Comment:
This is a bit confusing -- line 165 says it needs to have a deadline, line 168 says it doesn't, line 170 says that in some cases it doesn't.

Would it be clearer to say that:
The registration requirements may have a deadline for adding registered events.
If there is a deadline, it must not be earlier than the time registration is set to close.
If there is no deadline, competitors may add registered events up until that event takes place.